tests/gatys_et_al_2017/test_core.py

```diff
@@ -0,0 +1,50 @@
+import pytorch_testing_utils as ptu
+
+import pystiche_papers.gatys_et_al_2017 as paper
+from pystiche_papers.gatys_et_al_2017 import utils
+
+from pystiche.image.transforms.functional._resize import resize
+
+
+def test_gatys_et_al_2017_nst_smoke(subtests, mocker, content_image, style_image):
+    mock = mocker.patch(
+        "pystiche_papers.gatys_et_al_2017.core.default_image_pyramid_optim_loop"
+    )
+
+    paper.gatys_et_al_2017_nst(content_image, style_image)
+
+    args, kwargs = mock.call_args
+    input_image, criterion, pyramid = args
+    get_optimizer = kwargs["get_optimizer"]
+    preprocessor = kwargs["preprocessor"]
+    postprocessor = kwargs["postprocessor"]
+
+    with subtests.test("input_image"):
+        ptu.assert_allclose(input_image, resize(content_image, 800))
+
+    with subtests.test("criterion"):
+        assert type(criterion) is type(  # noqa: E721
+            paper.gatys_et_al_2017_perceptual_loss()
```

Comment:
```suggestion
        assert isinstance(criterion, type(paper.gatys_et_al_2017_perceptual_loss()))
```
I think that is more _pythonic_. Could you also address this below and in a follow-up PR also in https://github.com/pmeier/pystiche_papers/blob/aba7a3d55bc2697f52da78caffc79936a13cbe31/tests/gatys_ecker_bethge_2015/test_core.py#L31-L32 ?